tgui/packages/tgui/interfaces/ATM.tsx

```diff
@@ -157,9 +157,15 @@ export const AuthenticatedWindow = (props, context) => {
           />
           &nbsp;
           <Button
+            content="Withdraw Cash"
             icon="check"
             onClick={() => act('withdrawal', { funds_amount: withdraw })}
           />
+          <Button
+            content="Withdraw Charge Card"
+            icon="check"
```

Comment:
change this icon to something else, you can find all the icons in the font awesome website